### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Pixmy**

### CODE REVIEW

**Comment by Pixmy** [comment]
<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8"> 
	<title>Arturo Cabrera</title>

	<link rel="icon" href="https://cdn0.iconfinder.com/data/icons/flat-designed-circle-icon-2/1000/mouse.png">
	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"> 
	<link href="https://fonts.googleapis.com/css?family=Baloo+Bhai" rel="stylesheet"> 
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300" rel="stylesheet"> 
	<link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet"> 
	<link href="https://cdn.rawgit.com/konpa/devicon/master/devicon.min.css" rel="stylesheet" >
	<link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.css" rel="stylesheet" >

	 <style type="text/css">
			 	body{
		  margin: 0;
		  background-color: #ecf0f1;
		  overflow-x: hidden;
		}

		/*Header*/
		header{
		  z-index: 999;
		  background-color: #16a085;
		  position: fixed;
		  color: white;
		  opacity: 0.7;
		  padding: .5em;
		  transition: 0.3s ease;
		  width: 100%;
		  text-align: center;
		  font-family: 'Roboto', sans-serif;
		  font-size: 1em;
		  box-shadow: 0px 0px 15px 1px rgba(15,108,91,.7);
		}

		header a{
		  margin-left: 15px;
		  padding: .5em;
		  transition: 0.3s ease;
		}

		header a:hover{
		  cursor: pointer;
		  background-color: #2ecc71;
		  transition: 0.3s ease;
		}

		.container{
		  width: 100%;
		  height: 5em;
		  padding-top: 2em;
		}

		.hola{
		  text-align: center;
		  margin-top: 30px;
		  color: #2ecc71;
		  font-size: 1.3em;
		  font-family: 'Pacifico', cursive;
		}

		.yo{
		  background-color: #ecf0f1;
		  height: 12em;
		  background-image: url('https://s21.postimg.org/tf15umqk7/pixmy.png');
		  background-size: contain;
		  width: 12em;
		  margin: 0 auto;
		  margin-top: 10px;
		  border-radius: 50%;
		}

		.nombre{
		  text-align: center;
		  margin-top: 20px;
		  font-size: 1.3em;
		  font-family: 'Baloo Bhai', cursive;
		  color:#27ae60;
		  animation-delay: 2s;
		}

		.profesion{
		  text-align: center;
		  font-family: 'Open Sans', sans-serif;
		  font-size: 0.9em;
		}

		.bosque{
		  margin: 0 auto;
		  margin-top: 50px;
		  width: 90%;
		  height: 14em;
		  background-image: url('https://s15.postimg.org/vl86sue57/bosque.png');
		  background-size: contain;
		  background-repeat: no-repeat;
		  background-position: center;
		  bottom: 0;
		  animation-delay: 1s;
		}

		.nube1{
		  position: absolute;
		  top: 15%;
		  left: 0;
		  height: 2em;
		  width: 9em;
		  background-size: contain;
		  background-repeat: no-repeat;
		  background-position: center;
		  background-image: url('https://s16.postimg.org/oh0e39cmt/nube.png');
		}

		.nube2{
		  position: absolute;
		  top: 35%;
		  right: 0;
		  height: 1.5em;
		  width: 5em;
		  background-size: contain;
		  background-repeat: no-repeat;
		  background-position: center;
		  background-image: url('https://s16.postimg.org/oh0e39cmt/nube.png');
		}

		.sobremi{
		  background-color: white;
		  padding-top: 30px;
		  padding-bottom: 30px;
		}

		.yoDinseno{
		  background-color: #bdc3c7;
		  margin: 0 auto;
		  background-size: contain;
		  background-repeat: no-repeat;
		  background-position: center;
		  border-radius: 50%;
		  background-image: url('https://s18.postimg.org/6epqii4t5/dise_o.png');
		  height: 9em;
		  width: 9em;
		}

		.subTitle{
		  margin: 10px;
		  text-align: center;
		  font-size: 1.5em;
		  color: #2ecc71;
		  font-family: 'Baloo Bhai', cursive;
		}

		.texto{
		  width: 50%;
		  text-align: center;
		  margin: 0 auto;
		  font-size: 1.1em;
		  font-family: 'Open Sans', sans-serif;
		}

		.habilidades{
		  text-align: center;
		  margin-top: 90px;
		}

		.contacto{
		  margin-top: 70px;
		}

		table{
		  height: 5em;
		  margin: 0 auto;
		  margin-top: 20px;
		  width: 50%;
		}

		table i{
		  font-size: 10vmin;
		  color: #1abc9c;
		}

		table td{
		  border: 20px solid white;
		  text-align: center;
		  font-family: 'Baloo Bhai', cursive;
		}

		#email{
		  font-weight: bold;
		}

		/*Animaciones*/
		body.scrolled header {
		  opacity: 1.0;
		  transition: opacity .3s;
		}
		body.scrolled header:hover {
		  opacity: 1.0;
		  transition: opacity .2s;
		}


	 </style>


	

	 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
	 <script src="//cdn.jsdelivr.net/jquery.scrollto/2.1.2/jquery.scrollTo.min.js"></script>


	<script type="text/javascript">
	 	$(document).ready(function(){
		    $(".nube1").animate({left: "300px"},10000);
		    $(".nube2").animate({left: (($( window ).width()/2)+200)+"px"},7000);

		    $('#sobremiA').on('click', function(){
		        $('html,body').animate({scrollTop: $(".sobremi").offset().top});
		    });

		    $('#hab').on('click', function(){
		        $('html,body').animate({scrollTop: $(".habilidades").offset().top});
		    });

		    $('#cont').on('click', function(){
		        $('html,body').animate({scrollTop: $("#email").offset().top});
		    });
		});

		window.addEventListener('scroll', function () {
		  document.body.classList[
		    window.scrollY > 20 ? 'add': 'remove'
		  ]('scrolled');
		});

	</script>

</head>
<body>
	<header class="animated slideInDown">
		<a id="sobremiA">Sobre mí</a>
		<a id="hab">Habilidades</a>
		<a id="cont">Contacto</a>
	</header>

	<div class="container">
		<div class="hola">!Hola! Soy</div>
		<div class="yo animated bounceIn"></div>
		<div class="nombre animated rubberBand">Arturo Cabrera</div>
		<div class="profesion animated slideInUp">DiSEÑADOR Y PROGRAMADOR</div>
		<div class="bosque animated jello"></div>
		<div class="nube1"></div>
		<div class="nube2"></div>
		<div class="sobremi">
			<div class="yoDinseno"></div>
			<div id="sbm" class="subTitle">Sobre mí</div>
			<div class="texto">
				Me llamo Arturo Cabrera, soy del estado de Guanajuato y me encanta
				diseñar páginas web, me gusta dar esa primera impresión la cual 
				hace que el usuario se quede dentro de la página, además 
				me parece muy entretenido programar y llegar a una solución óptima
				ya sea individualmente o en equipo.
				<p><p>
				Soy un alumno recién egresado del Instituto Tecnológico Superior del Sur de Guanajuato, pero he tenido varias experiencias laborales, por ejemplo <a href="http://sicenet.itsur.edu.mx:8088/_public/views/inicio.aspx">SQLjudge</a>, además he trabajado en proyectos relacionados con entidades bancarias.
			</div>

			<div class="habilidades subTitle">Habilidades</div>
			<table>
				<tr>
					<td><i class="devicon-html5-plain"></i>
					<td><i class="devicon-css3-plain"></i>
					<td><i class="devicon-javascript-plain"></i>
					<td> <i class="devicon-csharp-plain"></i>
					<td> <i class="devicon-bootstrap-plain"></i>
				</tr>
				<tr>
					<td> <i class="devicon-android-plain"></i>
					<td><i class="devicon-jquery-plain-wordmark"></i>
					<td> <i class="devicon-mysql-plain-wordmark"></i>
					<td> <i class="devicon-oracle-original"></i> 
					<td><i class="devicon-java-plain-wordmark"></i>
				</tr>
			</table>
			<div class="contacto subTitle">Contacto</div>
			<div class="texto">
				Si usted está interesado en mi trabajo por favor contácteme vía email
				<p><span id="email">pixmy@outlook.com</span>
			</div>
		</div>
	</div>
</body>
</html>
